examples/wordpress/wordpress.jsonnet

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2018 Bitnami
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+// ```
+// kubecfg update wordpress.jsonnet
+//
+// kubecfg delete wordpress.jsonnet
+// ```
+
+local kube = import "lib/kube.libsonnet";
+local fe = import "frontend.jsonnet";
+local be = import "backend.jsonnet";
+
+kube.List() {
+  items_+: {
```

Comment:
Ouch, I'd forgotten that `items_` didn't just work with nested objects, sorry :slightly_frowning_face:  (I haven't used the `v1.List` wrapper for a long time).

An easy improvement is to drop all the `} + {` lines:
```javascript
kube.List() {
  items_+: {
    frontend_pvc: fe.frontend.pvc,
    frontend_configmap: fe.frontend.configmap,
    ...
  },
}
```

Another possible alternative is to just change the way `items` is calculated so that it does a recursive walk, but I'm not sure how "surprising" you want to make your example:
```javascript
local findObjs(top) = std.flattenArrays([
  if (std.objectHas(v, "apiVersion") && std.objectHas(v, "kind")) then [v] else findObjs(v)
  for v in kube.objectValues(top)
]);

kube.List() {
  items_+: {
    frontend: fe,
    backend: be,
  },
  items: findObjs(self.items_),
}
```

That improvement should probably also go into kube.libsonnet, but if we're doing it "properly" then it needs some more helpful error messages when it encounters an invalid object structure...